src/bootstrap/utils.ts

```diff
@@ -92,6 +97,9 @@ async function bootstrapState({
       keysPath,
       password,
       fs,
+      rootKeyPairChange: async (keyPairData: RootKeyPairChangeData) => {
```

Comment:
This is not necessary in the `bootstrapState` function. The agent isn't actually running during this point.

The `rootKeyPairChange` callback should be optional so it doesn't need to be passed in here.

Reply:
Fixed everywhere